.editorconfig

```diff
@@ -1,14 +1,106 @@
-# For ktlint configuration. Ref: https://ktlint.github.io/
+[*]
+charset = utf-8
+end_of_line = crlf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
```

Comment:
it was true, I think it should stay true

Reply:
Good catch!